dev-lang/erlang/erlang-21.0.5.ebuild

```diff
@@ -0,0 +1,220 @@
+"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+"
+
+S="${WORKDIR}/otp-OTP-${PV}"
+
+PATCHES=(
+		"${FILESDIR}/18.2.1-wx3.0.patch"
+		"${FILESDIR}/${PN}-20.3.2-dont-ignore-LDFLAGS.patch"
+		"${FILESDIR}/${PN}-add-epmd-pid-file-creation-for-openrc.patch"
+	)
+
+SITEFILE=50"${PN}"-gentoo.el
+
+# Taken from the upstream build script, build_otp
+erlang_create_lib_configure_in() {
+	local bootstrap_apps="erts lib/asn1 lib/compiler lib/hipelib/ic lib/kernel
+			 lib/parsetools lib/sasl lib/snmp lib/stdlib lib/syntax_tools"
+	local sdirs=
+	for lib_app in ${bootstrap_apps}; do
+		case "${lib_app}" in
+			lib/*)
+				if [[ -f "${lib_app}/configure.in" ]]; then
+					mv "${lib_app}/configure.in" "${lib_app}/configure.ac" || die
+					app=`echo "${lib_app}" | sed "s|lib/\(.*\)|\1|"`
+					sdirs="${sdirs}test ! -f ${app}/configure || AC_CONFIG_SUBDIRS(${app}/.)\n" || die
+				fi;;
+			*)
+			;;
+		esac
+	done
+
+	local sed_bootstrap="s%@BOOTSTRAP_CONFIGURE_APPS@%$sdirs%g"
+
+	sdirs=
+	for lib_app in lib/*; do
+		local is_bapp=false
+		for bapp in ${bootstrap_apps}; do
+			test "${bapp} != ${lib_app}" || { "${is_bapp}"=true; break; }
+		done
+		if [[ "${is_bapp}" = false ]] && [[ -f "${lib_app}/configure.in" ]]; then
+			mv "${lib_app}/configure.in" "${lib_app}/configure.ac" || die
+			app=`echo "${lib_app}" | sed "s|lib/\(.*\)|\1|"` || die
+			sdirs="${sdirs}    test ! -f ${app}/configure || AC_CONFIG_SUBDIRS(${app}/.)\n"
+		fi
+	done
+
+	local sed_non_bootstrap="s%@NON_BOOTSTRAP_CONFIGURE_APPS@%$sdirs%g"
+
+	rm -f lib/configure.in || die
+	sed "$sed_bootstrap;$sed_non_bootstrap" > lib/configure.ac < lib/configure.in.src || die "Failed to create lib/configure.ac"
+
+}
+
+# Taken from the upstream build script, build_otp
+erlang_distribute_config_helpers() {
+	local aclocal_dirs=". ./lib/erl_interface ./lib/odbc ./lib/wx ./lib/megaco"
+	local autoconf_aux_dirs="./lib/common_test/priv/auxdir ./lib/erl_interface/src/auxdir ./lib/common_test/test_server ./lib/wx/autoconf"
+
+	local aclocal_master="./erts/aclocal.m4"
+	local install_sh_master="./erts/autoconf/install-sh"
+	local config_guess_master="./erts/autoconf/config.guess"
+	local config_sub_master="./erts/autoconf/config.sub"
+
+	for dir in ${aclocal_dirs}; do
+		"${install_sh_master}" -m 644 -t "${dir}" "${aclocal_master}" || die
+	done
+
+	for dir in ${autoconf_aux_dirs}; do
+		"${install_sh_master}" -d "${dir}" || die
+		"${install_sh_master}" -t "${dir}" "${install_sh_master}" || die
+		"${install_sh_master}" -t "${dir}" "${config_guess_master}" || die
+		"${install_sh_master}" -t "${dir}" "${config_sub_master}" || die
+	done
+}
+
+src_prepare() {
+	default
+
+	# Determines which directories to recurse into with autoconf
+	erlang_create_lib_configure_in
+
+	# Move local autoconf files into the neccessary directories
+	erlang_distribute_config_helpers
+
+	java-pkg-opt-2_src_prepare
+
+	eautoreconf
+}
+
+src_configure() {
+	use wxwidgets && setup-wxwidgets
+
+	econf \
+		--disable-builtin-zlib \
```

Comment:
Please kill this backslash madness and move all the arguments into `local myconf=()` array instead.

Reply:
done